tendermint/src/lite/types.rs

```diff
@@ -7,14 +7,14 @@ use crate::Hash;
 use failure::_core::fmt::Debug;
 use std::time::SystemTime;
 
-/// TrustedState stores the latest state trusted by a lite client,
+/// `TrustedState` stores the latest state trusted by a lite client,
 /// including the last header (at height h-1) and the validator set
 /// (at height h) to use to verify the next header.
 pub trait TrustedState {
     type LastHeader: SignedHeader;
     type ValidatorSet: ValidatorSet;
 
-    /// Initialize the TrustedState with the given signed header and validator set.
+    /// Initialize the `TrustedState` with the given signed header and validator set.
```

Comment:
I'd suggest linkifying any types which are in-scope thusly:

```suggestion
    /// Initialize the [`TrustedState`] with the given signed header and validator set.
```

On nightly (which is used by https://docs.rs) these will generate internal links within the documentation.